pallets/tokens/src/lib.rs

```diff
@@ -325,12 +325,12 @@ decl_module! {
             token_id: TokenId,
             account_id: T::AccountId,
             value: Balance,
-        ) -> frame_support::dispatch::DispatchResult{
```

Comment:
does it makes sense to remove `DispatchResult` ?

Reply:
i think thats the default one if not specified manually